src/chrome/content/rules/Webstaurant_Store.xml

```diff
@@ -1,18 +1,21 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://webstaurantstore.com/ => https://webstaurantstore.com/: Cycle detected - URL already encountered: https://www.webstaurantstore.com/
-Fetch error: http://www.webstaurantstore.com/ => https://www.webstaurantstore.com/: Cycle detected - URL already encountered: https://www.webstaurantstore.com/
-	Some pages redirect to http.
+
+	Problematic hosts in *webstaurantstore.com:
+
+		- cdnimg (different content)
+		- cdnimg1 (different content)
+		- cdnimg2 (different content)
+		- cdnimg3 (different content)
 
 -->
-<ruleset name="The Webstaurant Store (partial)">
+<ruleset name="The Webstaurant Store">
 
 	<target host="webstaurantstore.com" />
+	<target host="cdn.webstaurantstore.com" />
+	<target host="chat.webstaurantstore.com" />
 	<target host="www.webstaurantstore.com" />
```

Comment:
`www.webstaurantstore.com` should come right after `webstaurantstore.com`

Reply:
Oh okay, I thought the sort order was alphabetical...

Reply:
`www` is an exception for the alphabetical order as it is very common